src/clj/core.clj

```diff
@@ -38,14 +40,15 @@
 (defroutes public-routes
   (GET "/" req (html-response
                  (if (auth/is-logged-in req)
-                   [:p (str "Hello, " (str (get-in req [:session :user :id])))]
+                   (let [user-git-id "12345"] ; TODO: "12345" must be replaced with (str (get-in req [:session :user :id]))
```

Comment:
Now that #130 is merged `(get-in req [:session :user :id])` would get "our" user id, the git id is still available via `(get-in req [:session :user :oauth-github-id])` 
Alternatively you could use the `get-user-by-id` db function instead of `get-user-by-git-id`

Reply:
> Now that #130 is merged `(get-in req [:session :user :id])` would get "our" user id, the git id is still available via `(get-in req [:session :user :oauth-github-id])` Alternatively you could use the `get-user-by-id` db function instead of `get-user-by-git-id`

i can't merge this branch because i got a lint error in question-view: unresolved var form/radio-button ....
maybe you can solve this

Reply:
Huh this is really weird. These function calls were introduced in #122 and when I reviewed it, they worked. The linter in the CI also didn't report any problem there.
But this is seemingly not the first time this issue came up: I found a configuration for our linter to ignore some of the `hiccup.form` functions in our `config.edn`
Maybe you can add the ones that are now causing the linter to fail?

Reply:
I just did some work on my PR and ran into the same problem. This commit https://github.com/pkoerner/expert-parakeet/pull/134/commits/5befb29d0c37e4224dec530b60168ff7b5a88646 fixed it for me.